contrib/kind.sh

```diff
@@ -116,6 +116,8 @@ usage() {
     echo "-gm  | --gateway-mode                 Enable 'shared' or 'local' gateway mode."
     echo "                                      DEFAULT: shared."
     echo "-ov  | --ovn-image            	      Use the specified docker image instead of building locally. DEFAULT: local build."
+    echo "-ovr  | --ovn-repo                    Specify the repository to build OVN from DEFAULT: https://github.com/ovn-org/ovn"
+    echo "-ovb  | --ovn-branch                  Specify the branch to build OVN from DEFAULT: main"
```

Comment:
imho these 2 flags are confusing (their short option name). Could we just have the long option name ?

Reply:
Ahh wait this is not suppose to be here, will remove the commit.